### PR TITLE
Fixes molotov cocktails

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Turns out molotov cocktails out of whatever you've made never turned out to work because the supposed fuel addition would go poof, so all you'd have every time you made a molotov is a dud that did nothing unless you manually filled it up with welding fuel.
Also cleaned up the code a very little bit
I dunno, too strong?
:cl:
 * bugfix: Molotov cocktails that you fabricate now work again.

### CODE REVIEW

**Comment by Irradiation** [comment]
>shove rag into bottle
>rag magically clears ALL reagents out of it and pulls 100u Welderfuel out of the ether

Truly magical shit, rags.

**Comment by boy2mantwicethefam** [comment]
That's because I dunno how it actually worked or what it's supposed to do, but I don't think it's supposed to be a dud. I could halve the fuel and then rebrand the change as content instead of bugfix

File: code/modules/reagents/reagent_containers/food/drinks.dm [hunk 1]
```diff
@@ -1422,13 +1422,15 @@
...
-		slot_flags = initial(dummy.slot_flags)
+		var/obj/item/weapon/reagent_containers/food/drinks/D = /obj/item/weapon/reagent_containers/food/drinks/molotov
+		molotov = initial(D.molotov)
+		flags = initial(D.flags)
+		name = initial(D.name)
+		smashtext = initial(D.smashtext)
+		desc = initial(D.desc)
+		slot_flags = initial(D.slot_flags)
+		reagents.clear_reagents()
+		reagents.add_reagent(FUEL, 100)
```
Comment (MadmanMartian) [inline code/modules/reagents/reagent_containers/food/drinks.dm:1433]: Why would this just magically add fuel to it?

Comment (SonixApache) [reply code/modules/reagents/reagent_containers/food/drinks.dm:1433]: what the actual fuck lmao

**Comment by SonixApache** [comment]
this would break the ability to use empty molotovs as harmless light sources so that's a big :-1: from me chief

**Comment by bathosbathtime** [comment]
>unless you manually filled it with welding fuel
>
what did he mean by this

**Comment by boy2mantwicethefam** [comment]
Meh

**Comment by Exxion** [comment]
For the record, you've always had to manually fill them

Also, i ded pls buff

**Pull request closed without merging**